app/routes/index.js

```diff
@@ -0,0 +1,14 @@
+import Route from '@ember/routing/route';
+import ENV from 'website-my/config/environment';
+
+const API_BASE_URL = ENV.BASE_API_URL;
+
+export default class IndexRoute extends Route {
+  model = async () => {
+    const response = await fetch(`${API_BASE_URL}/users/self`, {
+      credentials: 'include',
+    });
+    const userData = await response.json();
+    return userData.status;
```

Comment:
What if the API response doesn't contain `status`? Do you think we should have a default value?

Reply:
Yess it should have 🤔default value.

Reply:
Let's please add it then :)

Reply:
@rohan09-raj

Reply:
@rohan09-raj when are we doing this?

Reply:
If I am providing the default value then, if I open the reload the page it is set to that default value. I have searched for this, but couldn't find anything that would solve this issue.

Reply:
What's the code you're writing for doing that?

Reply:
I was not sure but I provided the default value of status before the async function.